INTERPRETING.md

```diff
@@ -26,8 +26,8 @@ code](https://tc39.github.io/ecma262/#sec-types-of-source-code).
 
 ### Test262-Defined Bindings
 
-The contents of the following files must be evaluated in the test realm's
-global scope prior to test execution:
+The contents of the following harness files must be evaluated in the test
```

Comment:
it may be better mentioning `raw` flag here as well, given "must" sounds like no exception.

Reply:
Good catch!